controllers/userController.js

```diff
@@ -0,0 +1,71 @@
+
+import * as UserModel from "../models/userModel";
+import AppError from "../errors/AppError";
+
+const logger = require('../utils/logger')('logController');
+
+const getUserInfo = async (req, res) => {
+  logger.log('debug', 'logIn: %j', req.body);
+  const { user } = req;
```

Comment:
To be honest we do not have this object attached to req. So your code won'r work.